Code/fbuild.bff

```diff
@@ -38,11 +38,11 @@ Compiler( 'Compiler-x64Clang' )
 }
 Compiler( 'Compiler-x64-Linux' )
 {
```

Comment:
In the future, I'd like to extend the config files for Linux so that you can easily switch between versions, but for now, given that it's not going to be the right version for everyone, I think I'd prefer to keep this at v4.9 since that is the version I use to make official releases.

Reply:
Noted on g++ version. The binary name needs to be changed though because invocation of gcc implies good-old C code in the file, not C++ code which causes compilation errors. As with *.i extension - some gcc version might or might not detect the proper language based on file content. It's safer to clearly force C++ mode - saves nerves.